UI/src/app/pages/mapping/components/dialog/dialog.component.html

```diff
@@ -0,0 +1,88 @@
+<mat-card class="dialog">
+  <mat-list #shoes>
+    <ng-container *ngFor="let comment of comments">
+      <ng-template 
+      [ngTemplateOutlet]="loadTemplate(comment)" 
+      [ngTemplateOutletContext]="{ $implicit: comment}">
+      </ng-template>
+    </ng-container>
+  </mat-list>
+
+  <input [(ngModel)]="value" [disabled]="isDisabled()" placeholder="Input comment" class="input input-add" (keyup.enter)="add()">
```

Comment:
prefer not to use several classes.
e.g. 'input-add' would be enough here.
Inside of scss you can use `@extend .classname;`

```scss
.input {
 // some styles for input
}
.input-add {
   @extend .input; //adds rules from .input to .input-add
   //some additional styles 
}
```